homeassistant/components/nut/config_flow.py

```diff
@@ -0,0 +1,143 @@
+"""Config flow for Network UPS Tools (NUT) integration."""
+import logging
+
+import voluptuous as vol
+
+from homeassistant import config_entries, core, exceptions
+from homeassistant.const import (
+    CONF_ALIAS,
+    CONF_HOST,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_PORT,
+    CONF_RESOURCES,
+    CONF_USERNAME,
+)
+from homeassistant.core import callback
+import homeassistant.helpers.config_validation as cv
+
+from . import PyNUTData, find_resources_in_config_entry, pynutdata_status
+from .const import DEFAULT_HOST, DEFAULT_NAME, DEFAULT_PORT, SENSOR_TYPES
+from .const import DOMAIN  # pylint:disable=unused-import
+
+_LOGGER = logging.getLogger(__name__)
+
+
+SENSOR_DICT = {sensor_id: SENSOR_TYPES[sensor_id][0] for sensor_id in SENSOR_TYPES}
```

Comment:
Can we have a default selection of sensors that we want to have enabled ?

Reply:
When I look at the device dump library (https://networkupstools.org/ddl/), its all over the board which each device provides so I think its something the user has to pick manually.